src/util/colour.h

```diff
@@ -32,7 +32,9 @@
 #define TERM_DIM        (16*1)
 #define TERM_BRIGHT     (16*2)
 
+bool term_init(bool tty_);
 const char *term_colour(int code);
+const char *term_intensity(int code);
 const char *term_normal();
 
 #endif
```

Comment:
Please do not commit .fuse/xyz.log?

Reply:
Yeah sorry, that was a mistake. I'm gonna add .fuse/ to gitignore